### ISSUE TITLE
Add a "model showcase" mode

### ISSUE THREAD

**Issue by nsarrazin** [huggingface/chat-ui#690]
We've had some pretty successful spaces that used chat-ui to showcase new models, for example [zephyr-chat](https://huggingface.co/spaces/HuggingFaceH4/zephyr-chat), [yi-34b chat](https://huggingface.co/spaces/01-ai/Yi-34B-Chat), [notus](https://huggingface.co/spaces/argilla/notux-chat-ui), etc.

We could cater more to model makers by tweaking the UI a bit to showcase a single model more (instead of the model picker), maybe add more customization option (logos, etc.) 

I think it would be great if chat-ui could become the default tool for showcasing new models on the hub. The idea is not fully formed but i'd like to start a discussion on the topic 😄

**Comment by julien-c** [comment]
good idea, what would you suggest in terms of UX tweaks for single model mode?
